### ISSUE TITLE
<ion-menu> does not open in ios mode (Ionic4)

### ISSUE THREAD

**Issue by ghybs** [ionic-team/ionic#14295]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ ] **3.x**
[x] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**

Setup an `<ion-menu>` with a mechanism to open it (like `<ion-menu-button>` or explicitly calling the menu's [`open()`](https://github.com/ionic-team/ionic/tree/v4.0.0-alpha.1/core/src/components/menu#open) method).
- On Android / `"md"` mode, the menu appears as "overlay" type as expected.
- On iOS / `"ios"` mode, nothing happens.

**Expected behavior:**

On both modes, the menu appears (as "push" type for iOS).

**Steps to reproduce:**

Try opening the menu, either using the `<ion-menu-button>` in the header toolbar, or using the provided "Open menu" button that calls the `<ion-menu>`'s `open()` method.

**Related code:**
- 4.0.0-alpha.1 mode="md": http://plnkr.co/edit/fI2RbafAdmyPWVYn0Rfj?p=preview (this works)
- 4.0.0-alpha.1 mode="ios": http://plnkr.co/edit/GOmqwcxEPUZVHgkVeRtY?p=preview (does not work)
- 0.1.5 mode="ios": http://plnkr.co/edit/NZCrtv2lubaRxeNStdjx?p=preview (works)

```html
<ion-button onclick="openMenu()">Open menu</ion-button>
```
```javascript
const menu = document.querySelector('ion-menu[menu-id="myMenu"]');

function openMenu() {
  menu.open();
}
```

**Other information:**

This works in iOS in `@ionic/core` version 0.1.5, but fails from 0.1.6 to 4.0.0-alpha.1 (current)

Note: in Plunker for some reason version 0.1.5 sometimes has troubles picking up the menu in "md" mode.

**Comment by codyburleson** [comment]
I have the same issue. It works on desktop, but when I open the PWA on my iPhone and try it in Safari, the menu doesn't slide out.

Published site: https://codyburleson.github.io/

My source code: https://github.com/codyburleson/codyburleson.pwa

**Comment by ghybs** [comment]
Hi,

This seems to have been fixed since `@ionic/core` version `4.0.0-alpha.6`:
- 4.0.0-alpha.6 in `"ios"` mode: https://plnkr.co/edit/wyf75ejqvxj0RCvIWfGp?p=preview
- 4.0.0-alpha.6 in `"md"` mode: https://plnkr.co/edit/lPClzK0sPUDatjNE0kcB?p=preview
- 4.0.0-alpha.7 in `"ios"` mode: https://plnkr.co/edit/edtQDsnI0I0ywv6Jmh12?p=preview
- 4.0.0-alpha.7 in `"md"` mode: https://plnkr.co/edit/QlNmq2wOnAvWMihAG5hS?p=preview

BTW the `<ion-menu-button>` seems to be troublesome now, but replacing it by `<ion-menu-toggle><ion-button>` seems to work fine, with the added functionality that it disappears when the menu is not active / always displayed (with `<ion-split-pane>`).

Speaking about `<ion-split-pane>`, the latter seems to crash Ionic in Firefox in `"md"` mode when an `<ion-menu-toggle>` button is used. I will open a separate ticket for that.

**Comment by ghybs** [comment]
The menu correctly opens even in iOS mode since alpha.6.
Considering this issue as solved.
